src/smc-hub/client.coffee

```diff
@@ -2001,7 +2002,7 @@ class exports.Client extends EventEmitter
             options =
                 plan     : mesg.plan
                 quantity : mesg.quantity
-                coupon   : mesg.coupon
+                coupon   : mesg.coupon_id
```

Comment:
can you make it "coupon: mesg.coupon" or "coupon_id:mesg.coupon_id" for consistency?

Reply:
The call to stripe takes it as `coupon : "theCodeString"` but I want to clarify in our code when it's just a `coupon_id` vs an entire `coupon` object. 

I could change it to only right before the stripe call.